deploy/aws/buildspec.yml

```diff
@@ -2,8 +2,8 @@ version: 0.2
 
 env:
   variables:
-    AWS_REGION: "us-east-1"
```

Comment:
I suggest you have two sets of files - one for aws, one for aws cn - don't overwrite